### ISSUE TITLE
No spinner shown when searching for domain

### ISSUE THREAD

**Issue by asemiankevich** [status-im/ens-usernames#56]
Blank screen shown for several seconds when searching for domain.
Spinner should be shown instead

![image](https://user-images.githubusercontent.com/28752642/46861482-72c8c800-ce1b-11e8-923c-7cda92e560f3.png)

**Comment by 3esmit** [comment]
I agree that this screen seems a bit too empty. Suggestions on what we could put on place of this blank box?
